src/ProjectManagement/Projects/PackagesConfigNuGetProject.cs

```diff
@@ -199,30 +248,25 @@ private void UpdateFullPath()
 
         private List<PackageReference> GetInstalledPackagesList()
         {
-            UpdateFullPath();
-            if (File.Exists(FullPath))
+            try
             {
-                try
+                var xml = GetPackagesConfig();
+
+                if (xml != null)
                 {
-                    var reader = new PackagesConfigReader(XDocument.Load(FullPath));
+                    var reader = new PackagesConfigReader(xml);
                     return reader.GetPackages().ToList();
                 }
-                catch (Exception ex)
-                {
-                    if (ex is System.Xml.XmlException ||
+            }
+            catch (Exception ex) 
+                when (ex is System.Xml.XmlException ||
                         ex is PackagesConfigReaderException)
-                    {
-                        throw new InvalidOperationException(string.Format(
-                            CultureInfo.CurrentCulture,
-                            Strings.ErrorLoadingPackagesConfig,
-                            FullPath,
-                            ex.Message));
-                    }
-                    else
-                    {
-                        throw;
```

Comment:
Why you are swallowing all exception instead of re-throwing? why not just to catch System.IO and XML exceptions here?